computer/revere-rhino-x86_64-2/setup.sh

```diff
@@ -1 +1,43 @@
-add setup.sh content
+#!/bin/bash
+
+echo Root password?
+read root_password
+
+echo Revere user password?
+read user_password
+
+echo Revere 2.4 GHz Pwd?
+read wifi_24GHz_password
+
+echo Revere 5 GHz Pwd?
```

Comment:
The password is the same for both, so reuse?

Reply:
fixed